CatBreeds/Model/CatImageHandler.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  CatImageHandler.swift
+//  CatBreeds
+//
+//  Created by Areeb Durrani on 4/10/25.
+//
+
+import Foundation
+
+struct Thumbnail: Decodable {
+    let source: String
+}
+
+struct Page: Decodable {
+    let thumbnail: Thumbnail
+}
+
+struct Query: Decodable {
+    let pages: [Page]
+}
+
+struct ImageResponse: Decodable {
+    let query: Query
+}
+
+func removeChar(from string: String, delimiter: Character) -> String {
+    var s = string
+    if s.contains(delimiter) {
+        s.removeSubrange(s.firstIndex(of: delimiter)!..<s.endIndex)
+    }
+    return s
+}
+
+func sanitizeInput(breed: String, addSuffix: Bool) -> String {
+    var toReturn = breed.replacingOccurrences(of: " ", with: "_")
+    if toReturn.contains("Persian") && toReturn.contains("Traditional") {
+        toReturn = "Traditional_Persian"
+    }
+    toReturn = removeChar(from: toReturn, delimiter: "(")
+    toReturn = removeChar(from: toReturn, delimiter: ",")
+    toReturn = removeChar(from: toReturn, delimiter: "[")
```

Comment:
Why can't you define the removeChar function to accept array of chars instead of doing it one by one